.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+<!-- This form is for bug reports and feature requests! -->
+
+**Is this a BUG REPORT or FEATURE REQUEST?**:
+
+> Uncomment only one, leave it on its own line: 
+>
+> /kind bug
+> /kind feature
+
+
+**What happened**:
+
+**What you expected to happen**:
+
+**How to reproduce it (as minimally and precisely as possible)**:
+
+
+**Anything else we need to know?**:
+
+**Environment**:
+- NBP version:

```

Comment:
What is NBP version?
Add Kubernetes version better
- Kubernetes version (use `kubectl version`):

Reply:
@huzhengchuan Thanks. I will update it.